binding.gyp

```diff
@@ -0,0 +1,30 @@
+{
+  "targets": [
+    {
+      "target_name": "nodereport",
+      "sources": [ "src/node_report.cc", "src/module.cc" ],
+      "include_dirs": [ '<!(node -e "require(\'nan\')")' ],
+      "conditions": [
+        ["OS=='linux'", {
+          "defines": [ "_GNU_SOURCE" ],
+          "cflags": [ "-g -O2 -std=c++11", ],
+        }],
+        ["OS=='win'", {
+          "libraries": [ "dbghelp.lib" ],
+          "dll_files": [ "dbghelp.dll" ],
+        }],
+      ],
+    },
+    {
+      "target_name": "install",
+      "type":"none",
+      "dependencies" : [ "nodereport" ],
+      "copies": [
+        {
+          'destination': '<(module_root_dir)',
+          'files': ['<(module_root_dir)/build/Release/nodereport.node']
+        }]
+    },
+  ],
+}
+
```

Comment:
^-- comment not addressed. js uses 4 SPC indents, c++ uses 2? I suggest pulling the eslint config from nodejs to keep some consistency of code, and make it easier for people to contribute consistent code to the project.